package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "boray",
     "description": "Boray (Buckets moray) client library",
-    "version": "0.1.0",
+    "version": "0.2.0",
```

Comment:
Guessing this repo needs a rebase on master. Current master is at 0.5.0.

Reply:
Yep, hadn't touched since Dec. Updated now but not yet tested with all the other changes since.